test/functional/feature_notifications.py

```diff
@@ -33,7 +39,8 @@ def setup_network(self):
                             "-blocknotify=echo > {}".format(os.path.join(self.blocknotify_dir, '%s'))],
                            ["-blockversion=211",
                             "-rescan",
-                            "-walletnotify=echo > {}".format(os.path.join(self.walletnotify_dir, '%s'))]]
+                            "-wallet={}".format(self.wallet),
+                            "-walletnotify=echo > {}".format(os.path.join(self.walletnotify_dir, '%w_%s' if os.name != 'nt' else '%s'))]]
```

Comment:
Instead of repeating the  `os.name != 'nt'` checks and `"{}_{}"` formatting throughout the test, maybe just define a helper function and call it here and below:

```python
def notify_output(walletname, txid):
    return txid if os == "nt" else "{}_{}".format(walletname, txid)
```

Reply:
I misunderstood what this code was doing when I wrote the suggestion above. But I guess you could still call the function here as `notify_outputname("%w", "%s")`

Reply:
Yes that's a neat suggestion, will also add a comment.

Reply:
Pushed amended commit https://github.com/ryanofsky/bitcoin/commit/4e9efac678a9c0ea4e4c7dd956ea036ae6cf17ec ([diff](https://github.com/ryanofsky/bitcoin/compare/56d2307446b3d91dd63a05a78d2ebe52edc5bcaf..4e9efac678a9c0ea4e4c7dd956ea036ae6cf17ec))